Makefile

```diff
@@ -624,12 +624,13 @@ bazel_coverage_test: failpoint-enable bazel_ci_simple_prepare
 		-//tests/globalkilltest/... -//tests/readonlytest/... -//br/pkg/task:task_test -//tests/realtikvtest/...
 
 .PHONY: bazel_build
+NOGO_FLAG=true
```

Comment:
Hi you can see in line 15 we `include Makefile.common`. And most of variables are set in `Makefile.common`. Can you move this one to `Makefile.common`?